templates/gameLog.html

```diff
@@ -59,7 +59,7 @@
         <div class="table-responsive">
           <table class="table table-striped table-bordered table-condensed box-score-team">
             <thead>
-              <tr><th>Name</th><th>Pos</th><th>Min</th><th>FG</th><th>3Pt</th><th>FT</th><th>Off</th><th>Reb</th><th>Ast</th><th>TO</th><th>Stl</th><th>Blk</th><th>PF</th><th>Pts</th></tr>
+              <tr><th>Name</th><th>Pos</th><th>Min</th><th>FG</th><th>3Pt</th><th>FT</th><th>Off</th><th>Reb</th><th>Ast</th><th>TO</th><th>Stl</th><th>Blk</th><th>PF</th><th>Pts</th><th>GmSc</th></tr>
```

Comment:
Add a tooltip like `<th title="Game Score">GmSc</th>`. Same thing in liveGame.html.